src/vs/workbench/workbench.main.ts

```diff
@@ -19,6 +19,9 @@ import 'vs/editor/browser/editor.all';
 // Menus/Actions
 import 'vs/platform/actions/browser/menusExtensionPoint';
 
+// External Explorers
+import "vs/platform/explorers/browser/explorerExtensionPoint";
```

Comment:
[needs change] Please move this file into the `workbench` folder, only shared services (between editor/workbench) and things around it should be in `platform`